autotest/ogr/ogr_mitab.py

```diff
@@ -2402,6 +2402,42 @@ def test_ogr_mitab_custom_datum_export():
     proj =  sr.ExportToMICoordSys()
     assert proj == 'Earth Projection 8, 9999, 42, 1, 2, 3, -4, -5, -6, -7, 0, "m", 15, 0, 0.9996, 500000, 0'
 
+###############################################################################
+# Check write/read description
+
+def test_ogr_mitab_description():
+    filename = '/vsimem/test_description.tab'
+    ds = ogr.GetDriverByName('MapInfo File').CreateDataSource(filename)
```

Comment:
the dataset should be destroyed at end of test

Reply:
Fixed